### PR TITLE
add code example for project credentials config

### PR DESCRIPTION
**Pull request by PhilippSpo**
Because I keep looking up how this works in the source code I figured this would help not only me!
Is this the most effective way of using the ProjectCredentialsConfig class?
Should we add a `catch` for potential errors?
@emmenko

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,6 +211,23 @@ By default the module will try to read the credentials from the following locati
...
 
+Example usage:
+```js
+import { SphereClient } from 'sphere-node-sdk'
+import { ProjectCredentialsConfig } from 'sphere-node-utils'
+
+const PROJECT_KEY = 'your-project-key'
+
+credentialsConfig = new ProjectCredentialsConfig()
+credentialsConfig._init()
```
Comment (emmenko) [inline README.md]: I think one of the most common use cases is this one (I never used it that way)
https://github.com/sphereio/sphere-stock-import/blob/master/src/coffee/run.coffee#L92-L98

Comment (hajoeichler) [inline README.md]: Yes, we use `enrichCredentials` in order to allow argument passing via command line, but also via config file.

**Comment by michael-s-grant** [comment]
Hello.  After spending some time figuring out how to specify the sphere project credentials (I had to dive into the source code to find out for the non-JSON form in the end; I didn't investigate the JSON form that closely), I thought it would be worth modifying the README.md to give the format for the JSON and non-JSON versions of the credentials files.  I thought of making a branch and introducing this myself, but then I saw this branch, which seems an appropriate place, but don't want to meddle with your team's branch.  Could I make a suggestion then that you (as a team) add this information to the README.md here?  Thanks!

**Comment by adnasa** [comment]
Great initiative @PhilippSpo 
Looking into the current version of `README`, the examples are in coffee.

We should continue with that until we decide to migrate to v2 (es6/es7)

**Comment by adnasa** [comment]
oh... 6 months old PR...
still.. my point stands 😄

**Comment by hajoeichler** [comment]
@michael-s-grant please create a PR with your suggestions. Thx.

**Comment by michael-s-grant** [comment]
@hajoeichler please give me write permission for this repository too and I will create a PR.

**Comment by hajoeichler** [comment]
@michael-s-grant done

**Pull request merged**